core/parallel_state_processor.go

```diff
@@ -395,12 +402,20 @@ func (p *ParallelStateProcessor) Process(block *types.Block, statedb *state.Stat
 	}
 
 	// Read requests if Prague is enabled.
-	var requests types.Requests
-	if p.config.IsPrague(block.Number()) && p.config.Bor == nil {
-		requests, err = ParseDepositLogs(allLogs, p.config)
+	var requests [][]byte
+	if p.config.IsPrague(block.Number()) {
```

Comment:
Should keep the check of bor here `p.config.Bor == nil`. Otherwise the following code will be executed after the HF.

Reply:
Good catch, will add the check back. Will also need it in state processor.